package.json

```diff
@@ -13,25 +13,22 @@
     "Lorenzo Stoakes <lorenzo@resin.io>"
   ],
   "scripts": {
-    "lint": "resin-lint angular.coffee core.coffee request.coffee test",
-    "pretest": "npm run lint && npm run-script prepublish",
+    "lint": "resin-lint test",
```

Comment:
This doesn't lint `core.ts`.
Also core.ts doesn't have any semicolon.
I was thinking that we agreed on using semicolons in typescript files, didn't we?
(I am 100% in favour of not using them)